concourse/pipelines/templates/dev_build_pipeline-tpl.yml

```
@@ -16,9 +16,9 @@ anchors:
         platform: linux
       image: ccp-7-image
       params:
-        GOOGLE_CREDENTIALS: ((google-service-account-key))
-        GOOGLE_PROJECT_ID: ((google-project-id))
-        GOOGLE_ZONE: ((google-zone))
+        GOOGLE_CREDENTIALS: ((ud/pxf/secrets/google-service-account-key))
```

```suggestion
        GOOGLE_CREDENTIALS: ((ud/pxf/secrets/ccp-ci-service-account-key))
```

For any of or CI tasks that use CCP for managing clusters, we should use `ccp-ci-service-account-key`